packages/design-system-react/src/components/text/Text.constants.ts

```diff
@@ -1,10 +1,9 @@
 import { TextVariant } from './Text.types';
 
-export const TEXT_CLASS_MAP: Record<TextVariant, string> = {
-  [TextVariant.DisplayLg]:
-    'text-[2.5rem] md:text-[4rem] font-bold md:font-medium',
+export const CLASSMAP_TEXT_VARIANT_FONTSTYLE: Record<TextVariant, string> = {
```

Comment:
Found and instance of `TEXT_CLASS_MAP` 
![Screenshot 2025-03-28 at 2 53 16 PM](https://github.com/user-attachments/assets/25529637-dd22-4086-9d31-84f60c6d8e1e)